guides/common/modules/proc_installing-and-configuring-the-puppet-agent.adoc

```diff
@@ -13,6 +13,17 @@ endif::[]
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite[]
 * The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
+The `puppet-agent rpm` is available on following repositories:
+** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 - `{RepoRHEL7ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 8 Server RPMs x86_64 - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 9 Server RPMs x86_64 - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
++
+To enable the required repository, enter:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
++
+----
+# subscription-manager repos --enable _My_Repository_Name_
+----
```

Comment:
@ehelms can you have a look? I think the explicit enablement is redundant if the user has synced the client repo and enabled it in a content view for the host.

Reply:
Given the prerequisites and how SCA tends to work these days I do think its redundant.

Reply:
I am not sure what changes you are suggesting. Could you please elaborate on the changes.

Reply:
The suggestion is to drop lines 21-26